### ISSUE TITLE
First item is focused when opening the user-switcher with the cursor

### ISSUE THREAD

**Issue by trevorjonesd2l** [Brightspace/user-switcher#14]
The focus is placed there so it's accessible to NVDA. Ideally it should be accessible by NVDA but also not focus the first item when selected with the cursor.

After some investigation it seems like NVDA button press looks like a cursor press.
